features/FxA-I-11/README.md

```diff
@@ -0,0 +1,128 @@
+# Match 1 - Control vs. No Show
+> Eye Password Experiment FXA-80
+
+Part of the [Eye Password Experiment](https://github.com/mozilla/fxa/blob/rfeeley/eye-password-emoji/features/FxA-80-eye-password-experiment/README.md)
+
+Idea in AHA: https://mozilla.aha.io/ideas/ideas/FXA-I-11
+
+## Outcomes
+
+The success of this feature is being able to answer the question:
+
+"Is the 'Show' password button useful in its current form?".
+"Does it reduce the number of errors the users make on the change password page?"
+
+## Assumptions
+
+- Users use this functionality on Android more.
+- We believe the "Show" password button is useful.
+
+## Problem Statement
+
+The change password flow can be improved in the current system.
+
+## Hypothesis
+
+We believe that removing the unmasking functionality from both new and old password fields for people changing their passwords will achieve more errors and bounces.
```

Comment:
nit: I think it'd be more motivating to frame the hypothesis in the positive, even though the experiment is disabling something.  e.g. along the lines of "We believe that the unmasking functionality helps users to reduce errors when entering their password.  We will know this is true if removing the feature produces an increase in bounces and errors".